howso/direct/client.py

```diff
@@ -1,7 +1,7 @@
 from concurrent.futures import Future, ThreadPoolExecutor
 from contextlib import contextmanager
 from copy import deepcopy
-from datetime import datetime
+from datetime import datetime, UTC
```

Comment:
The `UTC` export is only python 3.11+. Use `timezone.utc` instead.
```suggestion
from datetime import datetime, timezone
```

Reply:
Fixed, thank you